libr/util/file.c

```diff
@@ -216,6 +216,7 @@ R_API char *r_file_abspath(const char *file) {
 R_API char *r_file_path(const char *bin) {
 	char file[1024];
```

Comment:
we shouldnt be using fixed-size buffers, there's a MAXFILEPATH constant for this on most systems, except HURD where you have to use the heap.. i dont see the point on using the stack for this

Reply:
It is not my code but ok I can change it too.